resident/resident-service/src/main/java/io/mosip/resident/aspect/LoginCheck.java

```diff
@@ -58,6 +59,7 @@ public class LoginCheck {
 	private ResidentSessionRepository residentSessionRepository;
 	
 	@Autowired
+	@Lazy
```

Comment:
Is it introduced because of circular depdency? means this is not right approach,please break the circualr depedency

Reply:
Fixed.